### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by AnnMarieW** [astrowonk/demo_dash_bug#1]
Hey @astrowonk 

Nice MWE.  This is a known bug.  You can join the conversation and follow the progress here:
https://github.com/plotly/dash/issues/1252

**Comment by astrowonk** [comment]
Yeah I made the demo and repo and then was looking through the existing issues and found some overlap, and was figuring out which one (if any) to comment on. Thanks!

**Comment by astrowonk** [comment]
Should I be concerned that plotly/dash#1252 has been open for 2+ years and no one is assigned?

**Comment by AnnMarieW** [comment]
A comment on the forum and in the issue might help keep it on the radar.  Or maybe it's tricky and needs a sponsor.
